### DISCUSSION TITLE
Error output if no values are received with the option Summarize

### DISCUSSION THREAD

**Discussion by ToropovAV** [grafana/grafana-infinity-datasource#1042] [Q&A]
Hi. 
I am setting up a panel in Grafana to a data source that may or may not give values. My task is just to output the number of records. If my data source returns records then there is no problem, if my data source does not return data then I get an error:
error while performing the infinity query. error evaluating summarize expression. No parameter 'number' found. expression not found in fields. Not applying summarize expression
How can I get around this? For example, by adding a check, if the data is not received, output null.
Panel settings:
Type: JSON
Parser: Backend
Source: URL
Format: Data Frame
Method: GET
Summarize: count(number)
![2024-11-02_11-22](https://github.com/user-attachments/assets/1cefee95-2380-4520-90c0-7db098bf34aa)

**Comment by yesoreyeram** [comment]
Hiya. Sorry you are having issues with the summarize and no data. This was already reported and you can track in https://github.com/grafana/grafana-infinity-datasource/discussions/892
